unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenController.cs

```diff
@@ -49,18 +44,23 @@ public class LoadingScreenController : IDisposable
 
             tipsController = new LoadingScreenTipsController(view.GetTipsView());
             percentageController = new LoadingScreenPercentageController(sceneController, view.GetPercentageView(), commonDataStore);
+            timeoutController = new LoadingScreenTimeoutController(view.GetTimeoutView(), worldState, this);
 
             this.playerDataStore.lastTeleportPosition.OnChange += TeleportRequested;
             this.commonDataStore.isSignUpFlow.OnChange += OnSignupFlow;
             this.sceneController.OnReadyScene += ReadyScene;
             view.OnFadeInFinish += FadeInFinished;
+
+            // The initial loading has still a destination to set. We are starting the timeout for the
+            // websocket initialization
+            timeoutController.StartTimeout(new Vector2Int(-1, -1));
```

Comment:
What is (-1, -1)?

Reply:
Its an unexistent scene; Im using it to get the WebSocket timeout started; where no scene has been requested to load but still we have to check the websocket timeout.
As soon as a new scene gets requested, this value gets overriden.